### DISCUSSION TITLE
Choosing arduino pins for modules.

### DISCUSSION THREAD

**Discussion by EliasA97** [jgromes/RadioLib#669] [General]
Hello, I have a simple but noob question.

I'm making a board with SX1262 and the atsamd21 which is found on arduino zero. 
Should I pick whatever pins I like as long as they're interrupt ones or stick with the examples pins?

**Comment by jgromes** [comment]
The example pins are just that - an example, specifically for Arduino Uno, but you can pick whatever suits you, under the conditions that:

1. The SPI pins are connected to the SPI. It doesn't have to be the default SPI interface, RadioLib can use any SPI as long as it is supported by the Arduino core.
2. DIO1 is connected to a pin that supports external interrupts. This is limited on older platforms (such as the Arduino Uno), newer ones usually support external interrupts on any pin.
3. The other pins (NSS, BUSY and NRST) can be any digital pin.

**Reply by EliasA97** [reply]
Great, thank's!
